pulpcore/cli/container/distribution.py

```diff
@@ -132,7 +132,8 @@ def update(
             distribution_ctx.update(body={"repository": ""}, non_blocking=True)
             body["repository_version"] = f'{distribution["repository"]}versions/{version}/'
         elif distribution["repository_version"]:
-            repository_href, _, _ = distribution["repository_version"].partition("versions")
+            # unused x1 and x2 are to get us around a mypy/1.2 complaint about '_'
+            repository_href, x1, x2 = distribution["repository_version"].partition("versions")
```

Comment:
```suggestion
            repository_href, dummy1, dummy2 = distribution["repository_version"].partition("versions")
```

"_" is the python suggested way to do things like this...
Anyway, i always suspected this to collide with `_ = gettext.gettext`...
Your comment kind of drops out of its context when not viewd in the diff, but the resulting code....

Reply:
> "_" is the python suggested way to do things like this...

Oh, it's definitely "pythonic" - but it breaks ion mypy1.2. 

I think the comment is appropriate even w/out the context - "we don't use the 2nd and 3rd values from the tuple, but we can't use '_' because mypy 1.2 complains" feels like a long way to say the same thing. What did you have in mind?

Reply:
Just calling them "dummy", to make it obvious they are not used.
Next time someone tries to use `_` again, mypy will do the yelling for us.